app/helpers/show_summary_helper.rb

```diff
@@ -0,0 +1,40 @@
+# ShowSumamryHellper define handy methods such as check fields aviabaility
+module ShowSummaryHelper
+  def self.study_details?(presenter)
+    presenter.solr_document._source['participants_type_of_instruction_tesim'].nil? ||
+      presenter.solr_document._source['participants_type_of_instruction_tesim'][0] == '' ||
+      presenter.solr_document._source['participants_amount_of_instruction_tesim'].nil? ||
+      presenter.solr_document._source['participants_amount_of_instruction_tesim'][0] == '' ||
+      presenter.solr_document._source['summary_linguistictarget_tesim'].nil? ||
+      presenter.solr_document._source['summary_linguistictarget_tesim'][0] == ''
+  end
+
+  def self.participant_details?(presenter)
```

Comment:
@yf508 same issue as in my previous comment. e.g. this function returns true for a record with empty fields.

Reply:
It wont's surprise me for a record with empty fields. It **only** return false if all fields being checked are non empty.